packages/@ember/-internals/runtime/tests/system/object/computed_test.js

```diff
@@ -457,6 +457,28 @@ moduleFor(
       assert.deepEqual(n.normalized, []);
     }
 
+    ['@test @each works on array with falsy values'](assert) {
+      let obj = EmberObject.extend({
+        falsy: [null, undefined, false, '', 0, {}],
+        truthy: [true, 'foo', 123],
+
+        falsyComputed: computed('falsy.@each.foo', () => {
+          assert.ok(true, 'falsy computed');
+        }),
+
+        truthyComputed: computed('truthy.@each.foo', () => {
+          assert.ok(true, 'truthy computed');
+        }),
+      }).create();
+
+      // should throw no errors
+      obj.falsyComputed;
+
+      assert.throws(() => {
```

Comment:
Shouldn’t this be expectAssertion?

Reply:
Ah, correct, wonder why prod tests didn't catch that